launch/client.py

```diff
@@ -875,8 +880,15 @@ def batch_async_request(
                 f"Disallowed options {set(batch_task_options.keys()) - allowed_batch_task_options} for batch task"
             )
 
+        if not (bool(inputs) ^ bool(urls)):
```

Comment:
Also what are people's opinions on allowing a mix of urls/inputs for batch inference? I definitely don't think we need to do it now, but I'm ambivalent on in the future. Wonder if it makes sense to even have a mix of urls + inputs.

Reply:
I think I'd lean towards not allowing the mix. Seems esoteric IMO.